packages/components/src/color-picker/style.scss

```diff
@@ -227,6 +227,11 @@
 	padding-top: 16px;
 	display: flex;
 	align-items: flex-end;
+	min-width: 22em;
```

Comment:
Why do we use "em"s here?  It seems like we prefer pixels in general in Gutenberg?

Reply:
It is still possible to have the issuue on mobile 
<img width="320" alt="Capture d’écran 2020-07-27 à 11 21 09 AM" src="https://user-images.githubusercontent.com/272444/88531935-97476180-cffb-11ea-8fa7-4a16a52da854.png"> 

I  wonder if a better fix is to wrap inputs (return to the line) when they reach a certain size instead of increasing the popover size.

Reply:
> I wonder if a better fix is to wrap inputs (return to the line)

What do you mean by that?

> It seems like we prefer pixels in general in Gutenberg

Okay, I'll use pixels.

>Issue on mobile

Is this the min width we should make sure it's shown properly? `$break-mobile: 480px;`

Reply:
I've made the changes to be visible in mobile too with the minimum `min-width` required. 

> I wonder if a better fix is to wrap inputs (return to the line) when they reach a certain size instead of increasing the popover size.

@youknowriad just let me know what did you mean, so I can maybe explore a different approach.

If you meant though to listen to input changes and change layout or resize, I believe it would cause an unpleasant/confusing user experience.